app/posts/manage-training/2023-12-06-email-scheduling-in-the-admin-console.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Email scheduling in the admin console
```

Comment:
I'm never sure what to call the admin bit, but maybe we should avoid using "console" if that means something different to devs? 🤔 

```suggestion
title: Email scheduling in the admin interface
```

Reply:
We definitely need to come up with a better name, but this seems to be the most used for now.